tests/e2e/hci.go

```diff
@@ -65,6 +69,9 @@ var _ bool = ginkgo.Describe("hci", func() {
 		bootstrap()
 		scParameters = make(map[string]string)
 		remoteStoragePolicyName = GetAndExpectStringEnvVar(envStoragePolicyNameForHCIRemoteDatastores)
+		readVcEsxIpsViaTestbedInfoJson(GetAndExpectStringEnvVar(envTestbedInfoJsonPath))
```

Comment:
get list of esx using getHostsByClusterName method

Reply:
We need testbed info json in order to get nimbus pod , nimbus esx name and nimbus user to power off that particular host. Hence we can't use getHostsByClusterName method